tests/ccgbugs/tderefblock.nim

```diff
@@ -1,5 +1,5 @@
 discard """
-  cmd: "nim c -d:release -d:danger $file"
+  cmd: "nim c -d:release -d:danger --gc:orc $file"
```

Comment:
Would it be reasonable to test both ORC and non-ORC? There appeared to be at least somewhat distinct failure modes, and therefore, conversely, it's not obvious that ORC might not work with non-ORC failing.